frontend/app/routes/public/apply/$id/adult-child/children/$childId/cannot-apply-child.tsx

```diff
@@ -30,18 +32,20 @@ export const meta: Route.MetaFunction = mergeMeta(({ data }) => {
 
 export async function loader({ context: { appContainer, session }, params, request }: Route.LoaderArgs) {
   loadApplyAdultSingleChildState({ params, request, session });
+  const state = loadApplyAdultChildState({ params, request, session });
+  const { APPLICATION_YEAR_REQUEST_DATE } = getEnv();
+  const locale = getLocale(request);
 
   const t = await getFixedT(request, handle.i18nNamespaces);
 
   const meta = { title: t('gcweb:meta.title.template', { title: t('apply-adult-child:eligibility.cannot-apply-child.page-title') }) };
+  const currentDate = APPLICATION_YEAR_REQUEST_DATE ? new Date(APPLICATION_YEAR_REQUEST_DATE) : new Date();
+  const coverageStartDate = new UTCDate(state.applicationYear.coverageStartDate);
+  const formattedDate = coverageStartDate.toLocaleDateString(`${locale}-CA`, { year: 'numeric', month: 'long', day: 'numeric' });
```

Comment:
Considering using the convenience functions in `date-utils` to do this parsing and converting:
```suggestion
  const currentDate = APPLICATION_YEAR_REQUEST_DATE ? parseDateString(APPLICATION_YEAR_REQUEST_DATE) : new UTCDate();
  const coverageStartDate = parseDateString(state.applicationYear.coverageStartDate);
  const formattedDate = toLocaleDateString(coverageStartDate, locale);
```